nginx.conf

```diff
@@ -20,20 +20,29 @@
 		ssl_trusted_certificate	/srv/nginx/ca-certs.pem;
 		ssl_dhparam		/srv/nginx/dhparam.pem;
 
-		# Authentification
-		set			$setup_type ovf-ami;
```

Comment:
`setup_type ovf-ami` is being replaced in pmm-update: https://github.com/percona/pmm-update/blob/9bcd0f59bdb1e64914ec3133822086623cf84c6b/ansible/v011704/main.yml#L76-L81

Reply:
Reverted changes, so default setup type is now `ovf-ami`.